### ISSUE TITLE
Clarification on aria-sort="none"

### ISSUE THREAD

**Issue by zelliott** [w3c/aria-practices#1192]
Looking for some clarification on the usage of `aria-sort="none"`.

I have a data table with a number of columns. Some of the columns are sortable, some are not sortable. For the column that is sortable and being sorted on, I add `aria-sort="[ascending|descending]` appropriately.

For the columns that are sortable but not being sorted on, should I add `aria-sort="none"`? Or should I omit `aria-sort` altogether?

The Data Grid Example page (https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html) seems to imply that `aria-sort="none"` should be added "when a column is sortable but not sorted."

On the other hand, the W3C spec (https://www.w3.org/WAI/PF/aria/states_and_properties) states that each data table should only apply `aria-sort` to a single header at any given point in time.

Moreover, no SR (as far as I can tell) announces anything differently if a cell has `aria-sort="none"` vs. omits `aria-sort`.

Looking for some clarificiation here... thanks.

**Comment by JAWS-test** [comment]
Related: https://github.com/w3c/aria/issues/724 and https://github.com/w3c/aria/issues/582

Good would be:
* no aria-sort = column not sortable
* aria-sort=none = column sortable, but not sorted
* aria-sort=[other value] = column sorted
* several aria-sort per table are possible
* Hierarchy of sorting can also be specified (this is currently not possible).

**Comment by kolaps33** [comment]
@JAWS-test 
please what is status of the workitem? is there plan to implement aria-sort?, like is described in the comment from Oct 3, 2019.

Because I see aria-example
https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html#

Is recommending the same use role="none" when column is sortable but not sorted.
`aria-sort="none" ... applied when a column is sortable but not sorted.`

It would be great if JAWS could support it.

**Comment by JAWS-test** [comment]
@kolaps33 I am not from Freedom Scientific, I just work with JAWS on a daily basis. Therefore, I can't tell you when problems in JAWS will be fixed.
